recipes/eooffshore_ics_ccmp_v02_1_nrt_wind/recipe.py

```diff
@@ -0,0 +1,151 @@
+import dask
+import dask.array as da
+from datetime import datetime
+from metpy.calc import wind_direction, wind_speed
```

Comment:
@andersy005, despite my earlier comments on this PR, and the `blocked:...` label I added, I actually think if we move this `metpy` import _inside the body of the `ics_wind_speed_direction`_ function below, this will work.

Reply:
The reason being that the Dataflow worker image _does have metpy_, so the only issue is actually that the FastAPI application doesn't have `metpy` at recipe _parse time_. But if that import is within the function body (where IIUC it is used anyway), then it will not be imported at parse time.

Reply:
It's already in the environment used to run the recipe, so I think if the import is moved into the function body, it should just work.

Reply:
Hi @cisaacstern and @andersy005, the latest recipe commit now has the `metpy` import inside `ics_wind_speed_direction()`. I've tested it locally with version 0.9.1.